app/views/auth/failure.html.haml

```diff
@@ -0,0 +1,7 @@
+.inset
+  %h1 
+    Sign in problem
+
+  %h4
+    There was a problem signing you in. Please
+    = link_to 'Try signing in with MyUSA again', auth_path
```

Comment:
Perhaps lower-case the T in "Try" and add a period at the end of the sentence? </pedant>